### PR TITLE
sil_el_ethiopian_latin initial commit

### PR DESCRIPTION
**Pull request by rik-shaw**
This is an initial commit for a keyboard I am calling "SIL EL - Ethiopian Latin".  The intention is that it is able to key all Ethiopian Latin-based scripts so can act as a 'standard install'.  For certain languages that don't have optimized keying from this keyboard we will make custom keyboards for them, but this keyboard should be functionally able to produce all Latin-based special characters for Ethiopian languages.

### CODE REVIEW

File: sil_el_ethiopian_latin/source/sil_el_ethiopian_latin.kps [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...</Options><Buttons><Button><ButtonType>install</ButtonType><Caption>&amp;Install</Caption><URL></URL><Left>0</Left><Top>0</Top></Button><Button><ButtonType>cancel</ButtonType><Caption>Cancel</Caption><URL></URL><Left>0</Left><Top>0</Top></Button><Button><ButtonType>about</ButtonType><Caption>&amp;About...</Caption><URL></URL><Left>0</Left><Top>0</Top></Button></Buttons><Registry/><StartMenu><Folder></Folder><Items/></StartMenu><Info><Version URL="">1.2</Version><Name URL="">SIL EL - Ethiopian Latin</Name><Copyright URL="">© SIL Ethiopia</Copyright><Author URL="mailto:computer_ethiopia@sil.org">SIL Ethiopia</Author><WebSite URL="www.silethiopia.org">www.silethiopia.org</WebSite></Info><Files><File><Name>sil_el_ethiopian_latin.bmp</Name><Description>File sil_el_ethiopian_latin.bmp</Description><CopyLocation>0</CopyLocation><FileType>.bmp</FileType></File><File><Name>..\build\sil_el_ethiopian_latin.kmx</Name><Description>Keyboard EL - Ethiopian Latin</Description><CopyLocation>0</CopyLocation><FileType>.kmx</FileType></File><File><Name>welcome.htm</Name><Description>File welcome.htm</Description><CopyLocation>0</CopyLocation><FileType>.htm</FileType></File></Files><Strings/></Package>
```
Comment (mcdurdin) [inline sil_el_ethiopian_latin/source/sil_el_ethiopian_latin.kps]: readme needs to be a separate file with just the first paragraph from the Welcome, or similar. See comments on https://github.com/tavultesoft/keyboards/pull/38#discussion_r104269908

**Comment by rik-shaw** [comment]
@mcdurdin the readme.htm has been added and *some* ISO 639-3 codes have been added to the .kmn file.  However, I can't sort out what the acceptable BCP-47 codes would be (if you can provide a clean reference for it, it would be helpful) for the README.md so I have left it out for now :-(  I also used a similar tidy cleanup for the welcome.htm as used for the sil_ethiopic and sil_ethiopic_power_g keyboard maps.

**Comment by mcdurdin** [comment]
Thank you for the update! I am currently at a conference and will review as soon as I get a chance.

**Comment by mcdurdin** [comment]
Desktop download - https://secure.tavultesoft.com/keyman/downloads/keyboards/details.php?KeyboardID=713

**Pull request merged**